src/index.ts

```diff
@@ -334,6 +334,18 @@ export function addFunction(functionName: string, functionDescription: AddFuncti
   };
 }
 
+/**
+ *
+ * @param layer New layer name
+ * @param priority The lower priorities are rendered first
+ */
+export function addRenderingLayer(layer: string, priority: number) {
```

Comment:
why move the code in the root index.ts ?

Reply:
there aldeay is addFunction .. so I don't mind but let's write somewhere what we want to do...